### PR TITLE
Add simple stream example

### PR DESCRIPTION
**Pull request by shlomikushchi**
Simplest streaming example for users using StreamConn

### CODE REVIEW

**Review by umitanuki — APPROVED** [review]

File: examples/websocket_example.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+if __name__ == '__main__':
+    import logging
+
+    logging.basicConfig(format='%(asctime)s %(message)s', level=logging.INFO)
+    conn = StreamConn(
+            ALPACA_API_KEY,
+            ALPACA_SECRET_KEY,
+            base_url=URL('https://paper-api.alpaca.markets'),
+            data_url=URL('https://data.alpaca.markets'),
+            # data_url=URL('ws://127.0.0.1:8765'),
```
Comment (umitanuki) [inline examples/websocket_example.py:19]: Do you want to remove this?

File: examples/websocket_example.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        @conn.on(r'^status$')
+        async def on_status(conn, channel, data):
+            print('polygon status update', data)
+
+        @conn.on(r'^A*$')
+        async def on_second_bars(conn, channel, bar):
+            print('bars', bar)
+
+    @conn.on(r'^trade_updates$')
+    async def on_account_updates(conn, channel, account):
```
Comment (umitanuki) [inline examples/websocket_example.py:32]: Can you remove this? There is no meaningful updates in this channel right now.

File: examples/websocket_example.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    @conn.on(r'^AM\..+$')
+    async def on_minute_bars(conn, channel, bar):
+        print('bars', bar)
+
+    # @conn.on(r'^AM*$')
+    # async def on_minute_bars(conn, channel, bar):
+    #     print('bars', bar)
+
+    quote_count = 0  # don't print too much quotes
+    @conn.on(r'Q\..+', ['AAPL'])
```
Comment (umitanuki) [inline examples/websocket_example.py:44]: What about trades?

File: examples/websocket_example.py (hunk 1 above)
Comment (shlomikushchi) [reply examples/websocket_example.py:19]: yes

File: examples/websocket_example.py (hunk 2 above)
Comment (shlomikushchi) [reply examples/websocket_example.py:32]: yes

File: examples/websocket_example.py (hunk 3 above)
Comment (shlomikushchi) [reply examples/websocket_example.py:44]: added

**Pull request merged**